Makefile

```diff
@@ -161,21 +162,21 @@ verify: verify-edit verify-merged
 
 # Run validation queries on obi-edit and exit on error.
 .PHONY: verify-edit
-verify-edit: src/ontology/obi-edit.owl  $(EDIT_VIOLATION_QUERIES) | build
+verify-edit: src/ontology/obi-edit.owl $(EDIT_VIOLATION_QUERIES)
 	$(ROBOT) verify --input $< --output-dir build \
 	--queries $(EDIT_VIOLATION_QUERIES)
 
 # Run validation queries on obi_merged and exit on error.
 .PHONY: verify-merged
-verify-merged: build/obi_merged.owl $(MERGED_VIOLATION_QUERIES) | build
+verify-merged: build/obi_merged.owl $(MERGED_VIOLATION_QUERIES)
 	$(ROBOT) merge \
 	--input $< \
 	verify \
 	--output-dir build \
 	--queries $(MERGED_VIOLATION_QUERIES)
 
 .PHONY: test
-test: verify
+test: | build/robot.jar verify
```

Comment:
Should remain `test: verify`

Reply:
Ok - for some reason the build/robot.jar wasn't being found so I had to throw that in there. Should I move it to -
```
verify: | build/robot.jar verify-edit verify-merged
```
?